app/scripts/abos/list/aboslist.html

```diff
@@ -29,6 +29,14 @@ <h3 class="panel-title"><span class="navbar-brand"><span class="small">{{abo.id}
              {{abo.vertrieb.beschrieb| translate}}
           </div>
         </div>
+        <div class="row vertrieb" ng-if="abo.vertrieb.beschrieb">
+          <div class="col-md-4 col-sm-5 col-xs-5" >
+            <span translate>Beschreibung:</span>
```

Comment:
`Beschreibung:` is too generic in my opinion. What do we describe? So maybe `Abo-Beschrieb:` (to be almost as short)

Reply:
Hmm... You marked this as resolved but the changed tackles the other remark I made...

Reply:
ups, I got confuse with the 2 requests. I reopen and apply the change

Reply:
done